script.js

```diff
@@ -1,12 +1,363 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+// Getting elements from the DOM 
+const displayShows = document.getElementById('shows-page-wrapper');
+const displayEpisodes = document.getElementById('episodes-page-wrapper');
+const rootShows = document.getElementById("root-shows");
+const rootEpisodes = document.getElementById("root-episodes");
+const navLink = document.getElementById('navigation-link');
+const searchBar = document.querySelector(".search-bar");
+const searchForEpisodes = document.querySelector(".episodes-search-bar");
+const parentDiv = document.getElementsByTagName('div');
+
+//Declaring variables for accessability
+let data = [];
+let parsedData = [];
+
+//Getting data from API and passing it as a parameter
+const loadShows = async () => {
+  try {
+    const response = await fetch('https://api.tvmaze.com/shows');
+    data = await response.json();
+    console.log(data);
+    allShows(data);
+    showSearch(data)
+  } catch (err){
+      console.error(err);
+    }
+   
+}
+
+const loadEpisodes = async () => {
+  try {
+    const response = fetch("https://api.tvmaze.com/shows/"+showID+"/episodes");
+    parsedData = await (await response).json();
+    console.log(parsedData);
+    episodesPage(parsedData);
+    episodeSearch(parsedData);
+  } catch (err) {
+    console.error(err);
+  }
+    
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+// A function to extract data and populate the webpage. 
+function allShows(obj) {
```

Comment:
I would probably rename this to populateShows as it is more descriptive of what it does